### PR TITLE
ResourceTiming: Add some scenarios to iframe RT test

### PR DESCRIPTION
**Pull request by noamr**
In support of https://github.com/whatwg/html/pull/7531
Test iframe RT reporting for cross-origin iframes.

### CODE REVIEW

File: resource-timing/resources/iframe-with-delay.sub.html [hunk 1]
```diff
@@ -1,3 +1,10 @@
 <!DOCTYPE html>
 <meta charset="utf-8">
 <img src="/xhr/resources/delay.py?ms={{GET[delay]}}" />
+<script>
+    window.addEventListener('load', () => {
+        window.parent.postMessage(
+            {navigationEntry: performance.getEntriesByType('navigation')[0].toJSON(),
```
Comment (yoavweiss) [inline resource-timing/resources/iframe-with-delay.sub.html:7]: Is the top level frame already checking these numbers? Or is this something you're planning to add in future tests?

**Review by yoavweiss — APPROVED** [review]
LGTM

File: resource-timing/resources/iframe-with-delay.sub.html (hunk 1 above)
Comment (noamr) [reply resource-timing/resources/iframe-with-delay.sub.html:7]: Oops, I see some files are missing from the commit. Will revise.

Comment (noamr) [reply resource-timing/resources/iframe-with-delay.sub.html]: @yoavweiss re-review? Added the actual tests.

File: resource-timing/cross-origin-iframe.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+<script src="resources/entry-invariants.js"></script>
+<script src="resources/observe-entry.js"></script>
+</head>
+<body>
+<body>
+<script>
+  const {REMOTE_ORIGIN} = get_host_info();
+
+  promise_test(async t => {
+    const delay = 500;
```
Comment (yoavweiss) [inline resource-timing/cross-origin-iframe.html:16]: `delay` is unused here and below

**Review by yoavweiss — APPROVED** [review]
LGTM % nit

**Pull request merged**